system-tests/_scripts/launch-cluster.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+if [ -z "${INSTALLER_URL}" ]; then
+  echo "Error: Please specify the INSTALLER_URL environment variable"
+  exit 1
+fi
+
+if [ -z  "${SSH_KEY_PATH}" ]; then
+  echo "Error: Please specify the SSH_KEY_PATH environment variable"
+  exit 1
+fi
+
+# Create cluster config
+cat << EOF > config.yaml
+---
+launch_config_version: 1
+deployment_name: dcos-ui-system-test-$(date +%s)
+installer_url: ${INSTALLER_URL}
+platform: aws
+provider: onprem
+aws_region: us-west-2
+aws_key_name: default
+ssh_private_key_filename: ${SSH_KEY_PATH}
+os_name: cent-os-7
+instance_type: m4.large
+num_masters: 1
+num_private_agents: 1
+num_public_agents: 1
+dcos_config:
+    cluster_name: DC/OS UI System Integration Tests
+EOF
+
+if ! dcos-launch create; then
```

Comment:
Do you think it makes sense for clarity reasons to specify the full path to the config (`-c`) and the cluster info (`-i`)?

I would do something like:

```sh
CLUSTER_CONFIG=/tmp/dcos-ui-cluster.conf
CLUSTER_INFO=/tmp/dcos-ui-cluster.info

cat <<EOF > $CLUSTER_CONFIG
...
EOF

if ! dcos-launch create -c $CLUSTER_CONFIG -i $CLUSTER_INFO; then
...
```

Reply:
Also, be **extra careful** with what goes to STDOUT. Pipe everything to `/dev/null` if you must, but the only string that should be echoed on STDOUT is the path to the cluster!

For example, does `dcos-launch` display progress messages?

Reply:
Yeah, will adjust the script accordingly.